istio/oidc-authservice/base/envoy-filter.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: networking.istio.io/v1alpha3
 kind: EnvoyFilter
 metadata:
   name: authn-filter
+  namespace: istio-system
```

Comment:
this is controlled by kustomize overlays.

Reply:
Ok makes sense. How come not with https://github.com/kubeflow/manifests/blob/1bb77f3f28ce901947d80161f8e6ff83cafbfa1d/kfdef/generic/auth_oidc/envoy-filter.yaml#L4 ?

Reply:
I think someone forgot to remove it?  @kkasravi

Reply:
@ryandawsonuk the manifests you mention are not built through kustomize.go. These are included in the platform specific code - bypassing kustomize's namespace feature.

/cc @gabrielwen